### PR TITLE
Webserver

### PR DESCRIPTION
**Pull request by audunstrand**
please give feedback, I have never done this befre

### CODE REVIEW

File: src/main/kotlin/no/nav/helse/App.kt [hunk 1]
```diff
@@ -4,47 +4,65 @@ import com.ibm.msg.client.jms.JmsConnectionFactory
...
 
 private const val DEFAULT_PORT = 1414
 
 fun main(args: Array<String>) {
-    App(System.getenv()).run()
-}
+   GlobalScope.launch { App(System.getenv()).run() }
+   embeddedServer(Netty, 8080) {
+      routing {
+         get("/isAlive") {
```
Comment (davidsteinsland) [inline src/main/kotlin/no/nav/helse/App.kt]: path is specified in lowercase (`isalive`) in naiserator.yaml

File: src/main/kotlin/no/nav/helse/App.kt [hunk 2]
```diff
@@ -4,47 +4,65 @@ import com.ibm.msg.client.jms.JmsConnectionFactory
...
+            call.respond(HttpStatusCode.OK)
+         }
+         get("/isReady") {
```
Comment (davidsteinsland) [inline src/main/kotlin/no/nav/helse/App.kt]: path is specified in lowercase (`isready`) in naiserator.yaml

File: src/main/kotlin/no/nav/helse/SykemeldingCounter.kt [hunk 3]
```diff
@@ -1,26 +1,32 @@
...
-            LOG.trace("Received {}", xml)
+   while (!Thread.interrupted()) {
 
-            try {
-                if (containsSykmelding(xml)) {
-                    // TODO: increment sykemeldingCount
-                }
-            } catch (ex: Exception) {
-                LOG.error("Error while receiving XML message", ex)
+      val symemeldinger = Counter.build()
```
Comment (davidsteinsland) [inline src/main/kotlin/no/nav/helse/SykemeldingCounter.kt]: variable should be declared outside `while`

File: src/main/kotlin/no/nav/helse/SykemeldingCounter.kt [hunk 4]
```diff
@@ -1,26 +1,32 @@
...
+         .name("sykemeldinger").help("Antall sykemeldinger.").register()
+      val message = consumer.receive(15000)
+      if (message != null) {
+         val xml = message.getBody(String::class.java)
+         LOG.trace("Received {}", xml)
+
+         try {
+            if (containsSykmelding(xml)) {
+               symemeldinger.inc()
             }
```
Comment (davidsteinsland) [inline src/main/kotlin/no/nav/helse/SykemeldingCounter.kt]: would it be interesting to count messages that are not  `sykemelding`?

File: src/main/kotlin/no/nav/helse/SykemeldingCounter.kt (hunk 3 above)
Comment (jksolbakken) [reply src/main/kotlin/no/nav/helse/SykemeldingCounter.kt]: symemeldinger -> sykemeldinger

File: src/main/kotlin/no/nav/helse/SykemeldingCounter.kt [hunk 5]
```diff
@@ -0,0 +1,32 @@
+package no.nav.helse
+
+import io.prometheus.client.Collector
```
Comment (jksolbakken) [inline src/main/kotlin/no/nav/helse/SykemeldingCounter.kt]: ubrukte imports

File: src/main/kotlin/no/nav/helse/SykemeldingCounter.kt (hunk 4 above)
Comment (audunstrand) [reply src/main/kotlin/no/nav/helse/SykemeldingCounter.kt]: Maybe, if it is we can add it

**Pull request merged**